plugins/bap/utils/config.py

```diff
@@ -0,0 +1,66 @@
+"""Module for reading from and writing to the bap.cfg config file."""
+
+import os
+import idaapi
+
+cfg_dir = idaapi.idadir('cfg')
+cfg_path = os.path.join(cfg_dir, 'bap.cfg')
+
+def get(key, default=None, section='default'):
+    """Get value from key:value in the config file."""
+    from bap.utils import bap_comment
+    if not os.path.exists(cfg_path):
+        return default
+    with open(cfg_path, 'r') as f:
+        current_section = 'default'
+        for line in f.read().split('\n'):
+            if len(line) == 0:  # Empty line
+                continue
+            elif line[0] == ';':  # Comment
+                continue
+            elif line[0] == '.':  # Section
+                current_section = line[1:]
+            elif section != current_section:
+                continue
+            elif line.split()[0] == key:
+                return line.split()[1]
+    return default
+
+def set(key, value, section='default'):
```

Comment:
Yes, we need this since we have the functionality for the user to have a prompt on the first usage of the plugin, and this will set the configuration file. Since we have a copy-paste based system, we either need to do it like this, or give the user instructions saying "create a file in .... with the bap path at ..." etc. which I do not think is good user experience.